src/Univ2LpOracle.sol

```diff
@@ -138,19 +138,31 @@ contract UNIV2LPOracle {
     function wdiv(uint x, uint y) internal pure returns (uint z) {
         z = add(mul(x, WAD), y / 2) / y;
     }
-    // Compute the square root using the Babylonian method.
-    function sqrt(uint y) internal pure returns (uint z) {
-        if (y > 3) {
-            z = y;
-            uint x = y / 2 + 1;
-            while (x < z) {
-                z = x;
-                x = (y / x + x) / 2;
-            }
-        } else if (y != 0) {
-            z = 1;
+
+    // FROM https://github.com/abdk-consulting/abdk-libraries-solidity/blob/master/ABDKMath64x64.sol#L687-L709
```

Comment:
nit: should probably link to a specific commit hash instead of `master`, which can change